src/System.Threading.Tasks.Extensions/src/System/Runtime/CompilerServices/AsyncBuilderAttribute.cs

```diff
@@ -5,14 +5,20 @@
 namespace System.Runtime.CompilerServices
 {
     /// <summary>
-    /// Indicates the async method builder that should be used by the C# compiler to build
-    /// the attributed type when used as the return type of an async method.
+    /// Indicates the async method builder that should be used by a language compiler to
+    /// build the attributed type when used as the return type of an async method.
     /// </summary>
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Interface, Inherited = false, AllowMultiple = false)]
-    internal sealed class AsyncBuilderAttribute : Attribute
+    public sealed class AsyncBuilderAttribute : Attribute
```

Comment:
Actually, for my last question, I'm wondering the same thing about WinRT's `IAsyncOperation<T>` etc. Can/will these be made task-like to become valid return types of `async` methods (without having to construct wrappers), and - if so - where's the attribute applied (the .winmd file?) and where will the referenced builder type live (given it's .NET specific, it seems weird for a .winmd file to have an outgoing reference to a .NET specific type in its metadata)?